packages/client/src/comp/util/Switchable.tsx

```diff
@@ -1,35 +1,35 @@
 import React from 'react'
 import { SmallButton } from './Button'
+import { SignatureType } from './Signature'
 
 
-export type Choice = 'upload' | 'canvas'
-
 interface Props {
-  children: (choice: Choice) => React.ReactNode
+  children: (usedCanvas: SignatureType) => React.ReactNode
   visible: boolean
+  signatureType: SignatureType
+  setSignatureType: (_: SignatureType) => void
 }
 
-export const Switchable: React.FC<Props> = ({children, visible}) => {
-  const [choice, setChoice] = React.useState<Choice>('upload')
+export const Switchable: React.FC<Props> = ({children, visible, signatureType, setSignatureType}) => {
   return <>
     <div style={{display: visible ? 'flex' : 'none' , justifyContent: 'center'}}>
       <SmallButton
         style={{marginRight: '0', borderRadius: '4px 0 0 4px'}}
         color='primary'
-        disabled={choice === 'upload'}
-        onClick={() => setChoice('upload')}
+        disabled={signatureType === 'upload'}
+        onClick={() => setSignatureType('upload')}
       >
         Upload
       </SmallButton>
       <SmallButton
         style={{marginLeft: '0', borderRadius: '0 4px 4px 0'}}
         color='primary'
-        disabled={choice === 'canvas'}
-        onClick={() => setChoice('canvas')}
+        disabled={signatureType === 'canvas'}
+        onClick={() => setSignatureType('canvas')}
       >
         Pad
       </SmallButton>
     </div>
-    {(choice && children) && children(choice)}
+    {(children) && children(signatureType)}
```

Comment:
@marcoacfilho: I think we always want to render this child but not render the Switchable buttons if there isn't a child.  Can you confirm that I got this right for the experiment?